tsflex/features/segmenter/strided_rolling.py

```diff
@@ -250,20 +250,69 @@ def apply_func(self, func: FuncWrapper) -> pd.DataFrame:
         # expression only once, whereas a list comprehension evaluates its expression
         # every time).
         # See more why: https://stackoverflow.com/a/59838723
-        out = np.array(
-            list(
-                map(
-                    func,
-                    *[
-                        [
-                            sc.values[sc.start_indexes[idx]: sc.end_indexes[idx]]
-                            for idx in range(len(self.index))
-                        ]
-                        for sc in self.series_containers
-                    ],
+        out: np.array = None
+        if func.vectorized:
+            # Vectorized function execution
+
+            ## IMPL 1
+            # out = np.asarray(
```

Comment:
can you elaborate why you did not use this implementation

Reply:
Because of this
![image](https://user-images.githubusercontent.com/18898740/159446795-0f4f2ae8-4259-422f-bcda-e71230315d1e.png)

![image](https://user-images.githubusercontent.com/18898740/159446860-f31cf759-a35e-447b-a16f-637a89ac44dd.png)